apps/mobile/src/components/ContextSwitchMenu/ContextSwitchMenu.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import { Animated, TouchableOpacity, Text, StyleSheet, View, FlatList } from 'react-native'
+import FastImage from 'react-native-fast-image'
+import { Globe, Plus, CircleHelp } from 'lucide-react-native'
+import { map, sortBy } from 'lodash/fp'
+import useCurrentUser from '@hylo/hooks/useCurrentUser'
+import useCurrentGroup from '@hylo/hooks/useCurrentGroup'
+import { PUBLIC_GROUP } from '@hylo/presenters/GroupPresenter'
+import useChangeToGroup from 'hooks/useChangeToGroup'
+
+export default function ContextSwitchMenu () {
+  const [{ currentUser }] = useCurrentUser()
+  const [{ currentGroup }] = useCurrentGroup()
+  const changeToGroup = useChangeToGroup()
+  const myGroups = [PUBLIC_GROUP].concat(sortBy('name', map(m => m.group, currentUser.memberships)))
```

Comment:
Wondering if you know what order the memberships come in from the query? I assume not already sorted alphabetically by group name... (I also changed this to lodash methods for brevity/clarity here. I generally prefer lodash sortBy for things like this, but don't care a lot about that. Just wanted to remind you of options.)

Reply:
I don't know. Just following what Tibet set in the GlobalNav component on web